### PR TITLE
DATASTD 1574

### PR DESCRIPTION
**Pull request by markramonDL**
Grand Bend sample XML updated to 2021-2022 school year.

### CODE REVIEW

File: Samples/Sample XML/StudentIntervention.xml [hunk 1]
```diff
@@ -561,6 +561,6 @@
 				<InterventionPrescriptionIdentificationCode>255901-CheckAndConnect</InterventionPrescriptionIdentificationCode>
 			</InterventionPrescriptionIdentity>
 		</InterventionPrescriptionReference>
-		<BeginDate>2007-08-01</BeginDate>
+		<BeginDate>2017-08-01</BeginDate>
```
Comment (blmeyers) [inline Samples/Sample XML/StudentIntervention.xml]: Why only a 10 year jump here instead of 11 like the rest?

**Review by blmeyers — CHANGES_REQUESTED** [review]
Looks like LearningStandard in Standards.xml also needs to be updated, the CourseTitle is referencing 2009-2010, maybe worth dropping it rather than updating it?
Staff birth dates weren't affected, was that expected?

**Review by ejansson — COMMENTED** [review]
In this PR, I noted that 2010 and 2011 years have both been mapped to 2022. That might "degrade" some of the quality of the data if it was designed to represent a school year. I can't say that it will for sure - it is just a suspicion. Did you look into that or is there a reason for this choice?

File: Samples/Sample XML/AssessmentMetadata-ACT.xml [hunk 2]
```diff
@@ -130,7 +130,7 @@
 			<Title>ACT</Title>
 		</ContentStandard>
 		<AssessmentVersion>2005</AssessmentVersion>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/AssessmentMetadata-ACT.xml:132]: updated.

File: Samples/Sample XML/AssessmentMetadata-ACT.xml [hunk 3]
```diff
@@ -164,7 +164,7 @@
...
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/AssessmentMetadata-ACT.xml:166]: updated.

File: Samples/Sample XML/AssessmentMetadata-StateAssessment.xml [hunk 4]
```diff
@@ -116,13 +116,13 @@
...
 		<MaxRawScore>44.00000</MaxRawScore>
 		<AssessmentPeriod>
 			<AssessmentPeriod>uri://ed-fi.org/AssessmentPeriodDescriptor#EOY</AssessmentPeriod>
-			<BeginDate>2010-03-29</BeginDate>
-			<EndDate>2010-05-07</EndDate>
+			<BeginDate>2022-03-29</BeginDate>
+			<EndDate>2022-05-07</EndDate>
 		</AssessmentPeriod>
 		<Namespace>uri://ed-fi.org/Assessment/Assessment.xml</Namespace>
 	</Assessment>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/AssessmentMetadata-StateAssessment.xml:128]: updated the two remaining definitions.

File: Samples/Sample XML/AssessmentMetadata-StateAssessment.xml [hunk 5]
```diff
@@ -53,13 +53,13 @@
 			<Title>State Essential Knowledge and Skills</Title>
 			<PublicationYear>2011</PublicationYear>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/AssessmentMetadata-StateAssessment.xml:54]: updated.

File: Samples/Sample XML/AssessmentMetadata-StateAssessment.xml [hunk 6]
```diff
@@ -116,13 +116,13 @@
...
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/AssessmentMetadata-StateAssessment.xml:117]: updated.

File: Samples/Sample XML/Finance.xml [hunk 7]
```diff
@@ -11,7 +11,7 @@
 						<EducationOrganizationId>255901</EducationOrganizationId>
 					</EducationOrganizationIdentity>
 				</EducationOrganizationReference>
-				<FiscalYear>2011</FiscalYear>
+				<FiscalYear>2022</FiscalYear>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/Finance.xml:14]: Although the sample data set has some mismatched school year dates (2010-2011, and 2011-2012) the majority of it was 2010-2011 so I've used that as my guide to make the set consistent across entities for 2021-2022. Using the logic that the year was 2010-2011 I updated that Fiscal year to 2022.  Happy to talk some more about this if I'm missing something.

File: Samples/Sample XML/MasterSchedule.xml [hunk 8]
```diff
@@ -9,8 +9,8 @@
 		</SchoolReference>
 		<SessionReference>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/MasterSchedule.xml:10]: I noted this as well and left those identifiers alone. @vimayya , should we create a new ticket to work on this?

File: Samples/Sample XML/StaffAssociation.xml [hunk 9]
```diff
@@ -3326,7 +3326,7 @@
 		</EducationOrganizationReference>
 		<StaffClassification>uri://ed-fi.org/StaffClassificationDescriptor#Teacher</StaffClassification>
 		<PositionTitle>3rd Grade teacher</PositionTitle>
-		<BeginDate>2010-02-08</BeginDate>
+		<BeginDate>2021-02-08</BeginDate>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/StaffAssociation.xml]: updated the begin dates, hiring dates, and assignment dates.

File: Samples/Sample XML/StudentDiscipline.xml [hunk 10]
```diff
@@ -1475,7 +1475,7 @@
 	<DisciplineAction>
```
[the remark this answers is not in the document]
Comment (markramonDL) [reply Samples/Sample XML/StudentDiscipline.xml]: updated.

File: Samples/Sample XML/StudentIntervention.xml (hunk 1 above)
Comment (markramonDL) [reply Samples/Sample XML/StudentIntervention.xml]: not sure, but updated now.

**Comment by blmeyers** [comment]
All my review comments are addressed, and the data looks pretty solid.  @ejansson A lot of the updates that looked strange, when I dug in more were just fixing inconsistencies in the original data.  For example there's a mix of 2010-2011 and 2011-2012 data in some of the domains, so the general intent seems to have been to make sure the data is aligned to a 2021-2022 school year, making the dates make sense in general for representing a single school year.

At this point probably makes sense to spin off data standard tickets for updating the section and assessment identifiers to get away from dates, but this is good to go once Eric/Gabrielle are good with it.

**Comment by ejansson** [comment]
Ready to pull this in, but the merge is blocked pending some changes. Will also raise a ticket to remove school years from identifiers and course titles.

**Review by ejansson — APPROVED** [review]

**Review by blmeyers — APPROVED** [review]

**Pull request merged**